python/packages/ai/pyproject.toml

```diff
@@ -12,8 +12,8 @@ packages = [
 [tool.poetry.dependencies]
 python = ">=3.8,<4.0"
 botbuilder-core = "^4.14.4"
-aiohttp = "^3.8.5"
 semantic-kernel = "^0.3.7.dev0"
+aiohttp = "^3.8.5"
```

Comment:
(for my learning) why does this need to be moved?

Reply:
I was playing around with using the `openai` sdk so I removed it at some point but ended up readding it later on so it just got moved in that process.